src/Storefront/Resources/app/storefront/src/plugin/url-modal/url-modal.plugin.js

```diff
@@ -0,0 +1,84 @@
+import HttpClient from 'src/service/http-client.service';
+import Plugin from 'src/plugin-system/plugin.class';
+import PluginManager from 'src/plugin-system/plugin.manager';
+import PageLoadingIndicatorUtil from 'src/utility/loading-indicator/page-loading-indicator.util';
+import DeviceDetection from 'src/helper/device-detection.helper';
+import DomAccess from 'src/helper/dom-access.helper';
+import PseudoModalUtil from 'src/utility/modal-extension/pseudo-modal.util';
+
+/**
+ * This class extends the Bootstrap modal functionality by
+ * adding an event listener to modal triggers that contain
+ * a special "data-url" attribute which is needed to load
+ * the modal content by AJAX
+ *
+ * Notice: The response template needs to have the markup as defined in the Bootstrap docs
+ * https://getbootstrap.com/docs/4.3/components/modal/#live-demo
+ */
+export default class UrlModalPlugin extends Plugin {
+
+    static options = {
+        modalBackdrop: true,
+
+        urlAttribute: 'data-url',
```

Comment:
There should be no need for this option and can be used hardcoded. The registration is hardcoded, too 🤔

Reply:
I'd rather have this configurable to not stumble upon this in a situation where I have to change it and be angry about my own code 😅

Reply:
Yes, doesn't hurt to have this I think :man_shrugging: